testing/benchmark/Makefile

```diff
@@ -20,15 +20,20 @@ GOBENCH_HOST ?= http://localhost:9200
 GOBENCH_DEFAULT_TAGS = apm_server_version=$(APM_SERVER_VERSION)
 
 SSH_USER ?= ec2-user
-SSH_OPTS ?= -o LogLevel=ERROR -o StrictHostKeyChecking=no
+SSH_OPTS ?= -o LogLevel=ERROR -o StrictHostKeyChecking=no -o ServerAliveInterval=60 -o ServerAliveCountMax=10
 SSH_KEY ?= ~/.ssh/id_rsa_terraform
 WORKER_IP = $(shell terraform output -raw public_ip)
 
 SHELL = /bin/bash
 .SHELLFLAGS = -o pipefail -c
 
 # This profile will also be used by the Terraform provider.
-export AWS_PROFILE ?= default
+
+# export AWS_PROFILE if CI is not defined
+ifeq ($(CI),)
+	export AWS_PROFILE ?= default
+endif
```

Comment:
terraform was failing if `AWS_PROFILE` was defined but no corresponding profile could be found in config.

related issue: https://discuss.hashicorp.com/t/error-error-configuring-terraform-aws-provider-failed-to-get-shared-config-profile-default/39417/3